README.md

```diff
@@ -174,7 +187,7 @@ Type: `String`
 Optional: `true`
 Default: `null`
 
-Format the return value from the Sugarcoat `Promise`. By default, the expanded `config` object is returned. Options are `'json'` and `'html'`. The `'json'` option simply runs `JSON.stringify` on the expanded `config` object. This is useful when using the `--output` flag in the CLI.
+Format will allow you to change the return value from the Sugarcoat `Promise`. When used with the `--output` flag in the CLI, Sugarcoat will give you the format you selected(`'json'` or `'html'`) directly in your CLI. If you are running Sugarcoat as a module, the return value from the Sugarcoat `Promise` will be the format you selected(`'json'` or `'html'`). *Note: This is best when used with the CLI's `--output` flag.*
```

Comment:
I moved some of the wording around does that make it easier to understand? 
`Format will allow you to change the return value from Sugarcoat's `Promise`. The two options are `'json'` or `'html'`. When used with the `--output` flag in the CLI, Sugarcoat will return the format you selected directly in your CLI. If you are running Sugarcoat as a module, the return value from the `Promise` will be the selected format. *Note: This is best when used with the CLI's `--output` flag.*`